### ISSUE TITLE
Commenting a motion

### ISSUE THREAD

**Issue by tstelzer** [scrooloose/nerdcommenter#260]
Generally speaking *plugin X does Y* is not a good argument for implementing a feature, I know, however: I very much miss the ability to comment a motion, from tpopes commentary. Prepending a count helps in a way, but still lacks the precision I am used to in vim. Things like:

Comment out 3 lines up:
`gc3k`

... outer paragraph:
`gcap` 

... inner brackets
`gcib`

etc.

A workaround would be to select first, and comment out second, but a motion would be preferable.

Combining motions with the multitude of your plugins features would be brilliant indeed. Hope I am not beating a dead horse here, I haven't found a mention of motions in either the docs nor the issues.

*In either case, thanks for your great plugin, just found it and am very happy using it!*

**Comment by alerque** [comment]
I agree this would be a great thing to add. Basically it would just amount to a parallel set of functions that took a motion as a required argument and passed that range to the various comment functions rather than the current selection. You could then bind those functions however you liked. For many scenarios this would just be a reversal of the key sequence already available and one less character (`comment_binding<motion>` instead of `v<motion><comment_binding>`), but for some scenarios it would enable more flexibility because motion commands are repeatable in a way that selections are often not.

That being said the original author isn't actively developing this at all right now and I'm basically on the project as a community maintainer. Given my limited time, I'm unlikely to sit down and implement this feature anytime soon. If anybody want's to take a stab at it I will be happy to review and merge when it works.
